docs/getting-started/node-nvm.mdx

```diff
@@ -0,0 +1,60 @@
+---
+sidebar_position: 2
+---
+
+# NVM and Node.Js
+
+## Uninstall Node.Js(if you have it installed outside of nvm)
+
+Try to look for your current node.js file in `C:\Program Files` and uninstall it.
+
+
+## Installing the NVM 
+
+Head over to the [NVM Setup](https://www.freecodecamp.org/news/nvm-for-windows-how-to-download-and-install-node-version-manager-in-windows-10/) and follow all the guidelines and intructions
```

Comment:
Would be nice if the actual steps to install NVM are listed in this document as it seems directly related with the document itself.